app/views/account/organisations/edit.html.erb

```diff
@@ -0,0 +1,30 @@
+<% set_page_title(title_with_error_prefix(t('page_titles.account_organisation'), @organisation_form.errors.any?)) %>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_with(model: @organisation_form , url: account_organisation_path, method: :patch) do |f| %>
+      <% if @organisation_form &.errors.any? %>
+        <%= f.govuk_error_summary %>
+      <% end %>
+
+      <%= render DfE::Autocomplete::View.new(
+        f,
+        attribute_name: :organisation_id,
+        form_field: f.govuk_collection_select(:organisation_id,
+                                              Organisation.order(:slug),
+                                              :id,
+                                              :name,
```

Comment:
You've probably seen #1033 but flagging it just in case - I think it would make sense to update this to allow searching by abbreviation once that's merged. I'm happy for that to be in a subsequent PR though, if you want to get this one merged quicker.

Reply:
thanks, will do